api-mock/mocks/SessionMock.ts

```diff
@@ -273,6 +273,28 @@ export const SessionMock: Partial<typeof Session> = {
         });
         return deserializeSession(response);
     },
+    createNewSession: async (
```

Comment:
I don't think we need this.

Reply:
When I use the `createNewSessionWithoutRequestResponse` method that's already present, I don't see any set-cookie headers - possibly since there is no req/res object to add headers to
The session object has the expiry as a number, but we're concerned about the timezone that's sent in the header
```
  refreshToken: {
    createdTime: 1749703674308,
    expiry: 1758343674308,
    token: '...'
  },
```
vs
```
Expires=Fri, 12 Jun 2026 05:10:37 GMT
```